### ISSUE TITLE
flask server api for getting storied within specified date range

### ISSUE THREAD

**Issue by RMCoder198** [scorelab/fact-Bounty#166]
- [ yes] I am running the latest version
- [yes ] I checked to make sure that this issue has not already been filed

# Expected Behavior

There should be a api for getting storied within specified date range

# Current Behavior

currently , there is none for date range

**Comment by Tushar19varshney** [comment]
@RMCoder198 
Is there any requirement for this API in frontend?

**Comment by RMCoder198** [comment]
@ivantha @root5533  should i start working on it?

**Comment by RMCoder198** [comment]
@Tushar19varshney Yes, there should be but i will start working after getting confimation from mentors.

**Comment by Tushar19varshney** [comment]
@RMCoder198 I am not involving myself in your work but just asking that Is there any endpoint in frontend regarding date specified API?

**Comment by RMCoder198** [comment]
@Tushar19varshney There is no end point for date range in front-end yet. We can add it if mentors allows it.
